library/alloc/src/lib.rs

```diff
@@ -69,6 +69,7 @@
 #![warn(missing_docs)]
 #![warn(missing_debug_implementations)]
 #![allow(explicit_outlives_requirements)]
+#![allow(incomplete_features)]
```

Comment:
I'm torn here.  I would of course love to have these be `impl const`, but if it needs `incomplete_features` it feels like maybe core/alloc/std shouldn't be using it yet.

Maybe @m-ou-se would have thoughts here?

Reply:
I agree that this seems premature. The latest updates from lang team backlog bonanzas (as of several months ago) seem to suggest we are not yet committed to a specific behavior/path of implementation, so exposing (even unstably) const trait impls seems like it's not a good idea quite yet.